.scalafmt.conf

```diff
@@ -0,0 +1,11 @@
+version = "3.6.1"
+runner.dialect = scala213
+project.git = true
+docstrings.style = Asterisk
+docstrings.wrap = no
+assumeStandardLibraryStripMargin = true
+newlines.beforeCurlyLambdaParams = multilineWithCaseOnly
+align.tokens=[]
+align.openParenCallSite = false
+align.openParenDefnSite = false
+binPack.literalArgumentLists = true
```

Comment:
Why not keep the `maxColumn = 100` from scalacenter/bloop.

https://github.com/scalacenter/bloop/blob/f736b2a55a79307f4985ea93408d049466accb88/.scalafmt.conf#L3

Reply:
Ah, must of just missed it. I'll add it back in.